pkg/kubelet/kuberuntime/kuberuntime_gc.go

```diff
@@ -357,17 +357,17 @@ func (cgc *containerGC) evictPodLogsDirectories(allSourcesReady bool) error {
 					// See https://github.com/kubernetes/kubernetes/issues/52172
 					//
 					// We only remove unhealthy symlink for dead containers
-					klog.V(5).Infof("Container %q is still running, not removing symlink %q.", containerID, logSymlink)
+					klog.V(5).InfoS("Container is still running, not removing symlink.", "containerID", containerID, "logSymlink", logSymlink)
```

Comment:
```suggestion
					klog.V(5).InfoS("Container is still running, not removing symlink", "containerID", containerID, "symlink", logSymlink)
```

Reply:
should probably be `path` rather than `symlink`